journald/lib_journald.cpp

```diff
@@ -55,6 +56,10 @@ bool enabled();
 namespace mesos {
 namespace journald {
 
+// The number of characters in `{"key":"",value":""},` which is the
+// minimum size of any label when serialized into JSON.
+const Bytes LABEL_PADDING_SIZE = Bytes(21);
```

Comment:
You missed one quote before `value` and hence it is 22 bytes.

How about this instead:
```
const Bytes LABEL_PADDING_SIZE = Bytes(string("{\"key\":\"\",\"value\":\"\"},").size());
```